app/helpers/react_components/mentoring/mentor_discussion.rb

```diff
@@ -0,0 +1,47 @@
+module ReactComponents
+  module Mentoring
+    class MentorDiscussion < ReactComponent
```

Comment:
I think this can just be `Discussion`. We don't need the `Mentor` bit as it's namespaced.